### ISSUE TITLE
How to enable lang=postcss?

### ISSUE THREAD

**Issue by drnic** [jetthoughts/vuejs-rails-starterkit#67]
If I change app.vue's `<style>` section to choose `lang=postcss` I get an error when I refresh the home page:

```
Failed to compile.

./app/javascript/app.vue?vue&type=style&index=0&id=6fb8108a&lang=postcss&scoped=true& (./node_modules/vue-loader/lib??vue-loader-options!./app/javascript/app.vue?vue&type=style&index=0&id=6fb8108a&lang=postcss&scoped=true&) 18:2
Module parse failed: Unexpected token (18:2)
File was processed with these loaders:
 * ./node_modules/vue-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| 
| 
> p {
|   font-size: 2em;
|   text-align: center;
```

All I did was set the `lang` in app.vue:

```html
<style lang="postcss" scoped>
p {
  font-size: 2em;
  text-align: center;
}
</style>
```

**Comment by pftg** [comment]
Thanks, @drnic, I'll check this today

**Comment by pftg** [comment]
Hey @drnic, postcss is enabled by default, so you do not need to have explicit to say `lang="postcss"`. 

This error appears because in webpacker we do not have any rules which will handle posctss lang. But all css/scss automatically handled with postcss. If you will change postcss to css it will work outofbox. But if need to add new processor we have to update webpack config with it.

**Comment by pftg** [comment]
All postcss configurations are located in `postcss.config.js`, there you can change plugins and stage level.

**Comment by pftg** [comment]
@drnic is it a good enough solution for you or do you think need to change current behavior?

**Comment by drnic** [comment]
Thanks for looking into this. I might be confused about what to expect from postcss -- I first started using it inside pure vuejs apps to mash tailwindcss classes together.

I tried the following replacement app.vue but I'm not seeing "hello vue!" show in blue.

```html
<template>
  <div class="class3" id="app">
    <p>Hello Vue!</p>
  </div>
</template>

<script>
export default {};
</script>

<style scoped>
.class1 {
  font-size: 2em;
}
.class2 {
  color: blue;
}
.class3 {
  @apply class1 class2;
}
</style>
```

<img width="979" alt="image" src="https://user-images.githubusercontent.com/108/96830353-aca71d00-147e-11eb-92d5-5436d9b28b51.png">

If I change `#app` to class2 then it changes to blue; but with class3 there is no behavior. Ideas?
